README.md

```diff
@@ -141,6 +151,8 @@ This will produce a `build-win32` directory with the build products. Replace `mg
 - mgba/windows:w32
 - mgba/windows:w64
 
+If you wished to speed up the build process, consider adding the flag `-e MAKEFLAGS=-j4` to do a parallel build for mGBA with 4 cores instead of one core, if your CPU supports it. Change the `4` to however many cores you have on your build machine.
```

Comment:
Just say -jN where N is the desired number of cores. This is needlessly verbose now.

Reply:
I agreed. I simplified it. Thanks for the feedback.